jormungandr-lib/src/interfaces/mint_token.rs

```diff
@@ -55,6 +55,21 @@ impl<'de> Deserialize<'de> for TokenName {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct TokenIdentifier(identifier::TokenIdentifier);
 
+impl PartialOrd for TokenIdentifier {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        self.0
+            .token_name
+            .as_ref()
+            .partial_cmp(other.0.token_name.as_ref())
+    }
+}
+
+impl Ord for TokenIdentifier {
+    fn cmp(&self, other: &Self) -> std::cmp::Ordering {
+        self.0.token_name.as_ref().cmp(other.0.token_name.as_ref())
+    }
+}
```

Comment:
This is not entirely correct, because the hash of the minting policy can also be different in the future. Of course it is correct _for now_, so let's change it in a separate PR.

I would suggest you to:

1. Implement `PartialOrd` and `Ord` in chain-libs. You can simply use `derive(PartialOrd, Ord)`, which will compare fields in the order they are declared.
2. Just derive here as well.